README.md

```diff
@@ -69,93 +89,57 @@ _We have seen Kubernetes with cumbersome lifecycle management, high minimum syst
+<a name="super_quickstart"></a>
+## Super QuickStart
+To try k0s out on Linux (we recommend Amazon Linux 2 for quick experiments: use a t2.medium or larger VM for now), run the install script at (http://) *k0s.sh*, which identifies processor type, grabs the latest compatible k0s binary, saves it as _k0s_, and makes it executable.
 
 ```
-k0s server
+sudo curl -sSLf k0s.sh | sh
 ```
 
-This creates all the necessary certs and configs in `/var/lib/k0s/pki`. k0s runs all control plane components in separate "naked" processes, does not depend on kubelet or container engine.
+Then run k0s with the server option to create a k0s control plane, using the `--enable-worker` argument, which also starts and joins a worker node on the same machine.
 
-After control plane boots up, we need to create a join token for worker node:
-
-```
-k0s token create --role=worker
-```
-
-Join a new worker node to the cluster by running:
 ```
-k0s worker "superlongtokenfrompreviousphase"
+sudo k0s server --enable-worker
 ```
+Watch k0s start up for a few seconds (if issues, see the [troubleshooting guide](https://github.com/k0sproject/k0s/blob/main/docs/troubleshooting.md)), then open another terminal window (leaving k0s to percolate). In the new window, start by installing and validating kubectl:
 
-The client config (for kubectl or lens) can be found in `/var/lib/k0s/pki/admin.conf` on the controller. Replace `localhost` with the public IP address for the control plane.
-
-For more detailed description see [creating cluster documentation](docs/create-cluster.md).
-
-### k0s-in-docker
-
-**Note:** Running k0s, or any other Kubernetes distro, like this is not really a production ready setup. :)
-
-To run a single node controller+worker combo, just run it in docker with:
 ```
-docker run -d --name k0s-controller --hostname controller --privileged -v /var/lib/k0s -p 6443:6443 k0sproject/k0s
+curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl"
+chmod +x kubectl
+sudo mv ./kubectl /usr/local/bin/kubectl
+kubectl version
 ```
+Then retrieve the kubeconfig k0s has created for you in `/var/lib/k0s/pki/admin.conf`, and set up a shell variable that will let kubectl find and use it:
 
-Replace `<version>` with a released version number, we build the image for all tagged releases.
-
-That's it, in a minute or so there should be controller+worker combo running in the container.
-
-Just grab the kubeconfig with `docker exec k0s-controller cat /var/lib/k0s/pki/admin.conf` and paste e.g. into [Lens](https://k8slens.dev/). ;)
-
-Read more details at [running k0s in Docker](docs/k0s-in-docker.md).
-
-## Build
-
-`k0s` can be built in 3 different ways:
-
-Fetch official binaries (except `kine` and `konnectivity-server`, which are built from source):
 ```
-make EMBEDDED_BINS_BUILDMODE=fetch
+sudo cp /var/lib/k0s/pki/admin.conf ~/admin.conf
+export KUBECONFIG=~/admin.conf
```

Comment:
Is using the user's homedir here necessary? Maybe just use the CWD? I have a hunch there is or at least was some problem with expanding`~` in `KUBECONFIG`, maybe use `$HOME` and let the shell expand it.